frontends/web/src/components/darkmode/darkmode.tsx

```diff
@@ -15,10 +15,12 @@
  */
 
 import { useDarkmode } from '../../hooks/darkmode';
+import { setDarkTheme } from '../../api/darktheme';
 
 let darkmode: boolean | undefined;
 
 export const setDarkmode = (dark: boolean) => {
+  setDarkTheme(dark);
```

Comment:
If it's intentional to make it async, then it's fine. Otherwise, we should await this since it returns a promise. :)

Reply:
There is no return value and I'd proceed with the rest of the function flow even if the call would fail (which should not really happen since the only case is if there is a wrong/missing input param). I'd leave it as it is now

Reply:
ack. api call has no direct impact on the app - no point in awaiting :)